### DISCUSSION TITLE
Setting AutoWhitespace to false

### DISCUSSION THREAD

**Discussion by jcooper0** [teo-tsirpanis/Farkle#27] [Q&A] [answered]
Teo,

Thank you for the work on this library. 

Is there a simple change that can be applied to the calculator sample so that autoWhiteSpace is set to false?  It seems with autoWhiteSpace set to false, the calculator sample should parse "1+2" and should report an error with "1 + 2" However, when I add:

expression.AutoWhitespace false 

after the opScope setting, "1 + 2" parses without an error.

Thank you,
Jeff

**Comment by teo-tsirpanis** [comment, marked as answer]
Hello @jcooper0, thanks for using Farkle.

I didn't quite understand your problem, but if I am guessing correctly, you have added `.AutoWhitespace(false)` to `Farkle.Samples.CSharp.SimpleMaths`, then ran the `Farkle.Samples.Calculator` project, entered an expression with spaces, and it nevertheless succeeded?

In this case the behavior is expected, because the calculator is not using the C# sample project at all, but rather the _F#_ one. You should modify `SimpleMaths.fs` from the `Farkle.Samples.FSharp` project to change the behavior of the calculator.
